batchglm/train/numpy/base_glm/estimator.py

```diff
@@ -31,29 +32,63 @@ class EstimatorGlm(_EstimatorGLM, metaclass=abc.ABCMeta):
         A list of all log likelihood updates
     """
 
-    def __init__(
-        self,
-        model: ModelIwls,
-        input_data: InputDataGLM,
-        dtype: str,
-    ):
+    _train_loc: bool = False
+    _train_scale: bool = False
+    _model_container: BaseModelContainer
+    lls: List[float] = []
+    dtype: str = ""
+
+    def __init__(self, dtype: str, provide_batched: bool = False):
```

Comment:
What is `provide_batched`?

Reply:
sth. that is probably not supported anymore and a remnant of the old API @davidsebfischer ?

Reply:
Removed as it is nowhere used nor mentioned.